redirects-www.conf

```diff
@@ -260,11 +318,125 @@ rewrite ^/resources/about-fec/reports/firsttenyearsreport.pdf https://www.fec.go
+rewrite ^/info/LegislativeRecommendations([0-9]+) https://www.fec.gov/resources/cms-content/documents/legrec$1.pdf redirect;
+rewrite ^/info/presentations/(.*) https://www.fec.gov/help-candidates-and-committees/trainings/ redirect;
+rewrite ^/info/roundtable_materials/(.*) https://www.fec.gov/help-candidates-and-committees/trainings/ redirect;
+rewrite ^/info/roundtables/(.*) https://www.fec.gov/help-candidates-and-committees/trainings/ redirect;
+rewrite ^/info/state_outreach/(.*) https://www.fec.gov/help-candidates-and-committees/trainings/ redirect;
+
+# Broader redirects for /law/
+rewrite ^/law/cfr/ej_compilation/(.*) https://transition.fec.gov/law/cfr/ej_compilation/$1 redirect;
+rewrite ^/law/legrec([0-9]+) https://www.fec.gov/resources/cms-content/documents/legrec$1.pdf redirect;
+rewrite ^/law/legislative_recommendations_([0-9]+) https://www.fec.gov/resources/cms-content/documents/legrec$1.pdf redirect;
+
+# Broader redirects for /law/litigation/
+rewrite ^/law/litigation/(.*).pdf https://www.fec.gov/resources/legal-resources/litigation/$1.pdf redirect;
+rewrite ^/law/litigation/(.*) https://www.fec.gov/legal-resources/court-cases/ redirect;
+
+# Broader redirects for /legal-resources/
+rewrite ^/legal-resources/pdf/record/(.*).pdf https://www.fec.gov/resources/record/$1.pdf redirect;
+rewrite ^/legal-resources/court-cases/pdf/record/(.*).pdf https://www.fec.gov/resources/record/$1.pdf redirect;
+
+# Broader redirects for /members/
+rewrite ^/members/(.*).pdf /resources/about-fec/commissioners/$1.pdf redirect;
+rewrite ^/members/(.*).doc /resources/about-fec/commissioners/$1.doc redirect;
+
+# Broader redirects for /pages/ subdirectories
+rewrite ^/pages/brochures/(.*) https://transition.fec.gov/pages/brochures/$1 redirect;
+rewrite ^/pages/report_notices/(.*) http://transition.fec.gov/pages/report_notices/$1 redirect;
+
+# Broader redirects for /pdf/
+rewrite ^/pdf/ar([0-9]+) https://www.fec.gov/resources/cms-content/documents/ar$1.pdf redirect;
+rewrite ^/pdf/forms/(.*) https://www.fec.gov/resources/cms-content/documents/$1 redirect;
+rewrite ^/pdf/legrec([0-9]+) https://www.fec.gov/resources/cms-content/documents/legrec$1.pdf redirect;
+rewrite ^/pdf/nprm/(.*) /resources/legal-resources/rulemakings/nprm/$1 redirect;
+rewrite ^/pdf/record/(.*).pdf /resources/record/$1.pdf redirect;
+
+# Broader redirects for /press/
+rewrite ^/press/press([0-9]+)/(.*) https://www.fec.gov/resources/news_releases/$1/$2 redirect;
+rewrite ^/press/press(.*) http://transition.fec.gov/press/press$1 redirect;
+rewrite ^/press/archive/(.*) /resources/news_releases/$1 redirect;
+
+# Broader redirects for /pubrec/ and subdirectories
+rewrite ^/pubrec/(.*) https://transition.fec.gov/pubrec/$1 redirect;
+rewrite ^/pubrec/cfsdd/(.*) https://www.fec.gov/introduction-campaign-finance/how-to-research-public-records/combined-federalstate-disclosure-and-election-directory/ redirect;
```

Comment:
We need to be careful when rearranging the order of the broader redirects. We usually want the more specific one to win. So when you see nested directories like this: `/pubrec/cfsdd/`, that needs to be moved above the more broader redirect of `/pubrec/`

Reply:
Thanks for flagging @patphongs - fixed.